crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -1304,6 +1305,51 @@ impl<'w, 's> SystemParamFetch<'w, 's> for SystemChangeTickState {
     }
 }
 
+#[derive(Debug)]
+pub struct SystemName {
+    name: Cow<'static, str>,
+}
+
+impl SystemName {
+    pub fn name(&self) -> &Cow<'static, str> {
+        &self.name
+    }
+}
+
+impl<'a> SystemParam for SystemName {
+    type Fetch = SystemNameState;
+}
+
+// SAFETY: Only reads World entities
```

Comment:
I'm not sure this safety comment is correct.

I'm assuming this is copied from `Local`? If so, `Local` should also be fixed.